afl-plot

```diff
@@ -285,6 +291,8 @@ _EOF_
 
 ) | gnuplot
 
+echo "[?] You can also use -g flag to view the plots in an GUI window, and interact with the plots (if you have build afl-plot-ui). Run \"afl-plot\" to know more."
```

Comment:
typo, should be "if you have built", also switch "Run \"afl-plot\"" to "Run \"afl-plot -h\"". thanks